lib/src/display/stage.dart

```diff
@@ -354,8 +362,7 @@ class Stage extends DisplayObjectContainer {
 
   void materialize(num currentTime, num deltaTime) {
 
-    if (_stageRenderMode == StageRenderMode.AUTO ||
-        _stageRenderMode == StageRenderMode.ONCE) {
+    if (_stageRenderMode != StageRenderMode.STOP) {
```

Comment:
Same effect as the old condition, but also renders with the new AUTO_DIRTY mode.